scripts/auth-common.sh

```diff
@@ -13,16 +13,8 @@ fi
 
 set +x
```

Comment:
We don't need this since we're not using plain password anymore.

Reply:
Removed.